gammapy/estimators/utils.py

```diff
@@ -157,3 +155,72 @@ def estimate_exposure_reco_energy(dataset, spectral_model=None, normalize=True):
         reco_exposure = reco_exposure / ref_flux[:, np.newaxis, np.newaxis]
 
     return reco_exposure
+
+
+def _satisfies_conditions(info_dict, conditions):
```

Comment:
If this function is used only in `resample_energy_edges` you might as well define it inside the function.

Reply:
I commented before to move it outside, just because I thought `resample_energy_edges` was already complex enough...